src/components/SupportPage/SupportForm.jsx

```diff
@@ -0,0 +1,137 @@
+      />
+    );
+  }
+
+  renderSuccessMessage() {
+    return (
+      <StatusAlert
+        className="mt-3"
+        alertType="success"
+        iconClassName="fa fa-check-circle"
+        title="Request received"
+        message="The edX Customer Success team will contact you soon."
+        dismissible
+      />
+    );
+  }
+
+  render() {
+    const {
+      handleSubmit,
+      submitting,
+      invalid,
+      submitSucceeded,
+      submitFailed,
+      error,
+    } = this.props;
+
+    return (
+      <React.Fragment>
+        {submitFailed && error && this.renderErrorMessage()}
+        {submitSucceeded && this.renderSuccessMessage()}
+        <div className="support-form row">
+          <div className="col-12 col-md-6 col-lg-4">
+            <form onSubmit={handleSubmit(this.renderSuccessMessage)}>
+              <Field
+                name="emailAddress"
+                type="email"
+                component={RenderField}
+                label={
+                  <React.Fragment>
+                    Email Address
+                    <span className="required">*</span>
+                  </React.Fragment>
+                }
+                validate={[isRequired, isValidEmail]}
+                required
+              />
+              <Field
+                name="enterpriseName"
+                type="text"
+                component={RenderField}
+                label={
+                  <React.Fragment>
+                    Company
+                    <span className="required">*</span>
+                  </React.Fragment>
+                }
+                validate={[isRequired]}
+                required
+                disabled
+              />
+              <Field
+                name="notes"
+                type="text"
+                component={RenderField}
+                label="Notes"
+                validate={[maxLength512]}
+              />
+              <Button
+                type="submit"
+                disabled={invalid || submitting}
+                className="btn-primary"
+              >
+                <React.Fragment>
+                  {submitting && <Icon className="fa fa-spinner fa-spin mr-2" />}
+                  Contact Support
+                </React.Fragment>
+              </Button>
+            </form>
+          </div>
+        </div>
+      </React.Fragment>
+    );
+  }
+}
+
+SupportForm.defaultProps = {
+  error: null,
+};
+
+SupportForm.propTypes = {
+  // props From redux-form
+  handleSubmit: PropTypes.func.isRequired,
+  submitting: PropTypes.bool.isRequired,
+  invalid: PropTypes.bool.isRequired,
+  submitSucceeded: PropTypes.bool.isRequired,
+  submitFailed: PropTypes.bool.isRequired,
+  error: PropTypes.instanceOf(Error),
+
+  // custom props
+  match: PropTypes.shape({
+    url: PropTypes.string.isRequired,
+  }).isRequired,
+  initialValues: PropTypes.shape({ // eslint-disable-line react/no-unused-prop-types
```

Comment:
nit: see my other comment about the `initialValues` prop. Since we're passing this in, we should probably use the default values and then remove this `eslint-disable-line` comment.